src/types/Venue.ts

```diff
@@ -35,6 +36,7 @@ export interface Venue {
   parentId?: string;
   template: VenueTemplate;
   name: string;
+  access?: VenueAccessType[];
```

Comment:
Why is this an array?

Reply:
No idea. Should be a string.

Reply:
I haven't looked through the rest of the code to be sure, but I wonder if the original thought/intent here was that you could have multiple venue access modes at the same time? Does any of the rest of the code suggest that? (mostly want to make sure that we don't end up breaking the type when the rest of the code tries to use it like an array)